OpenRA.Mods.Common/Effects/GravityBomb.cs

```diff
@@ -72,9 +74,10 @@ public void Tick(World world)
 			velocity -= acceleration;
 			pos += velocity;
 
-			if (pos.Z <= args.PassiveTarget.Z)
+			var dat = world.Map.DistanceAboveTerrain(pos);
+			if (dat.Length <= 0)
```

Comment:
Actually, I think the original code was already correct here (assuming the PassiveTarget is not underground).